src/metax_api/models/data_catalog.py

```diff
@@ -60,3 +60,12 @@ def __repr__(self):
             self.catalog_json['research_dataset_schema'],
             self.catalog_json['dataset_versioning'],
         )
+
+    def user_has_access(self, request):
+        """
+        Overriding inherited operation to check permissions for datacatalogs
+        """
+
+        if request.method == 'GET' or request.user.is_service:
```

Comment:
Read-operations should probably cover more verbs than just GET. Elsewhere in Metax read-operations typically include GET, HEAD, and OPTIONS, for example in models/catalog_record.py: ``READ_METHODS = ('GET', 'HEAD', 'OPTIONS')``. Similarly in permissions/permissions.py.

I think permitting just GET might break browsable api too when no authentication is used (although that is not THAT important).